woocommerce/woocommerce-ios
# Zendesk: Registering for Push Notifications

### Details:
This PR wires the Zendesk Push Notifications Registration. Actual Zendesk Push handling isn't implemented here!

Ref. #393

### Testing: Authenticated
0. Make sure the Push Notifications Server is running
1. Log into your Store
2. Open the Support UI
3. File a new Ticket
4. Send the app to BG
5. Bug your favorite Happiness Engineer, and politely ask for a reply in your ticket

- [x] Verify a Push Notification comes thru

### Testing: Authenticated > Deauthenticated
0. Make sure the Push Notifications Server is running
1. With the app in the exact state as it was in the previous step, logout
2. Again, bug your favorite HE and get another reply in your ticket

- [x] Verify **NO** Push Notification comes thru
